src/icons/close-x.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+
+export const CloseIconX = ({ size = "16", color = "#000000" }) => (
```

Comment:
please create svg file like the rest of the files in icons

Reply:
Also, `react-icons` is already included in the project, maybe it's better to use them